plugins/transfer/plugin.go

```diff
@@ -176,7 +194,10 @@ type unpackConfiguration struct {
 
 func defaultConfig() *transferConfig {
 	return &transferConfig{
-		MaxConcurrentDownloads:      3,
-		MaxConcurrentUploadedLayers: 3,
+		MaxConcurrentDownloads:          9,
```

Comment:
Wondering why we decided to set this default in transfer service? The local (non-transfer service) client implementation has a different behavior where all layers are pulled in parallel if `WithMaxConcurrentDownloads` is not set.

Reply:
Reverting this, I wanted to change the default behaviour of containerd to always use parallel layer fetching with a safe default, but maybe this can be done in a subsequent PR.